crates/swc/tests/tsc-references/ES5For-of16.2.minified.js

```diff
@@ -1,5 +1,6 @@
 //// [ES5For-of16.ts]
 for(var _i = 0, _iter = []; _i < _iter.length; _i++){
     _iter[_i];
-    for(var _i1 = 0, _iter1 = []; _i1 < _iter1.length; _i1++)_iter1[_i1];
+    for(var _i1 = 0, _iter1 = []; _i1 < _iter1.length; _i1++);
```

Comment:
These tests changes are bugfix

Reply:
Previously we generated foo = `Expr::Seq(seq)` so logic expecting identifier in `foo` didn't work correctly.

Reply:
Now we generate `foo = seq.0` so it's analyzed/processed correctly